index.html

```diff
@@ -0,0 +1,70 @@
+<html>
+
+
+
+<main id="main">
```

Comment:
Please add the necessary html tags before and after the <html> tag. Because the browser used these tags to read your site code and also it helps for the visually impaired to navigate your website efficiently like so:
`<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>
<body>`